dist/placeholder-buster.txt

```diff
@@ -294,6 +294,21 @@ lmtonline.com###menuBar, #subHead:style(transform: none; -webkit-transform: none
 lmtonline.com###menuBar:style(margin-top: 50px !important;)
 lmtonline.com##.ctpl-fullbanner
 
+# https://github.com/NanoMeow/QuickReports/issues/3895
+marketscreener.com##.myFooter
+marketscreener.com##[id^="zpp"]
+
+# https://github.com/NanoMeow/QuickReports/issues/4266
+||engine.fxempire.com^
+||cdn.fxempire.com^
```

Comment:
I think these should be in EasyList.

Reply:
I've removed those two entries now, as explained below.